templates/includes/head.html

```diff
@@ -5,7 +5,7 @@
   <meta name="keywords" content="" />
   <meta name="author" content="Canonical webteam" />
   <meta name="viewport" content="width=device-width, initial-scale=1" />
-  <meta http-equiv="Content-Security-Policy" content="default-src 'self' 'unsafe-inline' https: http://www.google-analytics.com">
+  <meta http-equiv="Content-Security-Policy" content="default-src gap://ready file://* *; style-src 'self' http://* https://* 'unsafe-inline'; script-src 'self' http://* https://* 'unsafe-inline' 'unsafe-eval'">
```

Comment:
I'm not sure we want this to be so permissive. maas.io is just about to change to https and I don't think we want to allow any http source.